client/blocks/signup-form/index.jsx

```diff
@@ -194,9 +196,16 @@ class SignupForm extends Component {
 	/**
 	 * If the step is invalid because we had an error that the user exists,
 	 * we should prompt user with a request to connect his social account
-	 * to his existing WPCOM account
+	 * to his existing WPCOM account.
```

Comment:
Nit: "their" would be a better pronoun to use here rather than "his" :)

Reply:
> Nit: "their" would be a better pronoun to use here rather than "his" :)

@noahtallen Changed with b0dadf65de